pandapipes/std_types/std_type.py

```diff
@@ -48,6 +48,17 @@ def __init__(self, name, reg_par):
         """
         super(PumpStdType, self).__init__(name, 'pump')
         self.reg_par = reg_par
+        self._pressure_list = None
+        self._flowrate_list = None
+        self._reg_polynomial_degree = 2
+
+    def update_pump(self, pressure_list, flowrate_list, reg_polynomial_degree):
+        reg_par = regression_function(pressure_list, flowrate_list, reg_polynomial_degree)
+        self.res_par = reg_par
```

Comment:
do you really mean self.re**s**_par here?

Reply:
you are right. That is a type. Should be self.reg_par.